public/js/monitor.js

```diff
@@ -436,6 +489,50 @@ function resizeDashboardMargin() {
   }
 
   const dropDownMenu = document.querySelector(".mobile-nav.show-mobile");
+  const submitBtn = document.querySelector("#scan-user-email input[type='submit']");
+
   dropDownMenu.addEventListener("click", () => toggleDropDownMenu(dropDownMenu));
 
+  const acceptedFirstLocalesIncludesEnglish =
```

Comment:
nit: variable name `firstAcceptedLanguageIsEnglish` might be more descriptive.

Reply:
Revised name to be `acceptedFirstLanguageIsEnglish`